src/main/java/net/minecraftforge/network/MCRegisterPacketHandler.java

```diff
@@ -76,6 +76,16 @@ private Set<ResourceLocation> bytesToResLocation(byte[] all) {
             }
             return rl;
         }
+
+        /**
+         * Provides only the registered locations that the remote side has sent.
+         * This is useful for interacting with other modloaders via the network to inspect registered network ids.
+         *
+         * @return - The remote locations
+         */
+        public Set<ResourceLocation> getRemoteLocations() {
```

Comment:
I am fairly certain this method exposes the ability for modders to mess things up and modify the set of remote locations and also locations (as they are being set to the same instance)

Reply:
Good point, I will make a copy of it!